angular/src/lib/examples.module.ts

```diff
@@ -43,8 +43,8 @@ import { DatePickerExamplesModule } from './date-picker/examples/examples.module
     ToggleSwitchExamplesModule,
     EditableTextfieldExamplesModule,
     TopbarExamplesModule,
-    TimePickerExamplesModule,
-    DatePickerExamplesModule
+    // TimePickerExamplesModule,
```

Comment:
Meant to be commented out?

Reply:
Yes, we need a separate PR to fix the issue with these components.